.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull Request Template
+about: A change is gonna come - Sam Cooke
+title: ''
+labels:
+assignees: ''
+
+---
+
+## What's this PR do?
+A clear and concise description of what this PR stands for. You could also use lists to:
+
+- List new behaviors on existing components.
+- List new components added.
```

Comment:
I think is redundant to put new and added, if they were added it means that they are new. We could simply say 
`List added components.`